lichess-bot.py

```diff
@@ -1110,6 +1111,17 @@ def intro() -> str:
 auto_log_directory = "lichess_bot_auto_logs"
 
 
+def log_python_and_libraries() -> None:
+    """Log the installed libraries and the python version."""
+    logger.debug(f"Python version: {'.'.join(map(str, sys.version_info))}")
+    text = "Installed libraries:\n"
+    distributions = importlib.metadata.distributions()
+    for distribution in distributions:
+        text += f"{distribution.metadata['Name']}=={distribution.version}\n"
+    text += "\n"
+    logger.debug(text)
```

Comment:
Why do you create a single string instead of logging each line?

Reply:
So that the output looks like this:
```
DEBUG           Python version: 3.12.0-a.b
                requests==x.y.z
                chess==1.10.0
```
and not like this:
```
DEBUG           Python version: 3.12.0-a.b
DEBUG           requests==x.y.z
DEBUG           chess==1.10.0
```
I just think that it looks better and it's easier to unnderstand that all these lines are about the same thing.